Detectors/ZDC/calib/src/BaselineCalibSpec.cxx

```diff
@@ -115,6 +128,40 @@ void BaselineCalibSpec::sendOutput(o2::framework::DataAllocator& output)
   output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "ZDCBaselinecalib", 0}, info);         // root-serialized
   // TODO: reset the outputs once they are already sent (is it necessary?)
   // mWorker.init();
+
+  o2::zdc::CalibParamZDC& opt = const_cast<o2::zdc::CalibParamZDC&>(CalibParamZDC::Instance());
+  if (opt.rootOutput == true) {
+    mOutputDir = opt.outputDir;
+    if (mOutputDir.compare("/dev/null")) {
+      mHistoFileName = mOutputDir + fmt::format("{}_{}.root", fn, mRunNumber);
```

Comment:
this is error prone since mOutputDir is not guaranteed to have '/' in the end. Usually I do like
```
mHistoFileName = fmt::format("{}{}{}_{}.root", mOutputDir,  mOutputDir.back()=='/' ? "":"/" fn, mRunNumber);
```